desktop/cmp/grid/GridModel.js

```diff
@@ -207,24 +229,23 @@ export class GridModel {
     }
 
     /**
-     * Set sort by column
-     *
-     * This method is no-op if provided any sorters without a corresponding column
+     * This method is no-op if provided any sorters without a corresponding column.
+     * @param {(GridSorter|GridSorter[])} sorters - column(s) and direction for sorting.
```

Comment:
Should we have a convention for these typedefs (and also the callbacks) to be clear that they are only documentation time artifacts/names?  e.g. 'DGridSorter' /'D_GridSorter'

Reply:
Would you use `i` for interface, or does that seem too much like a reader should be able to find typescript lurking around somewhere?

The defs themselves are very useful in terms of structuring the docs w/o making the primary signature totally unreadable, so would like to keep using them - although in some cases I bet they will end up being placeholders for actual class implementations.